.github/workflows/ci-merge-checks.yml

```diff
@@ -0,0 +1,153 @@
+on:
+  pull_request:
+    types: [ready_for_review]
+    branches:
+      - develop
+      - release/**
+  workflow_dispatch:
+    # empty as it is used only to manually trigger the workflow
+  
+env:
+  CYPRESS_KEY: ${{ secrets.CYPRESS_KEY }}
+  NODE_VERSION: '16'
+  GH_TOKEN: ${{ github.token }}
```

Comment:
what’s the use for setting it globally GH_TOKEN 

I think $GH_TOKEN is even set to everyone’s personal account thingy too

Reply:
Agreed.  It looks strange or redundant, but this is required to use GH CLI from workflows.
See the first sentence in this page: 
https://docs.github.com/en/actions/using-workflows/using-github-cli-in-workflows